src/assets/apiLib/nodemailer.ts

```diff
@@ -0,0 +1,20 @@
+const nodemailer = require("nodemailer");
+
+const email = process.env.EMAIL;
+const pass = process.env.EMAIL_PASS;
+
+export const transporter = nodemailer.createTransport({
```

Comment:
we'll see what happens when you deploy. usually server-side stuff requires different deploys (usually paid). vercel might give you a free tier though or something like cold-starts. also you'll now have to keep this email/pass env vars updated in your deployed env as well and if sam changes it, it'll need to be updated. it would be simpler to use a free service like [this](https://web3forms.com/) to avoid these issues but this was good experience!